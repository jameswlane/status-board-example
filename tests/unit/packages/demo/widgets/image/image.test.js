const fs = require("fs");
const path = require('path');
var filePath = path.normalize(path.join(__dirname, '../../../../../../packages/demo/widgets/image/image.js'));

describe('Image Widget', function () {
  test('Example dummy test', function (done) {
    function callback(error, data) {
      if (error) {
        console.error("read error:  " + error.message);
      } else {
        var widget;
        eval(data);
        expect(widget.onData).toMatchSnapshot();
        done();
      }
    }

    fs.readFile(filePath, "utf8", callback);
  });
});