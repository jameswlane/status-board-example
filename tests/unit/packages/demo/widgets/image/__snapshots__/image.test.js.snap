// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Widget Example dummy test 1`] = `[Function]`;
